cosmosdb/management_endpoint.bal

```diff
@@ -850,12 +863,12 @@ public client class ManagementClient {
         return finalStream;
     }
 
-    # Perform queries on offer resources.
+    # Performs queries on offer resources.
     #
     # + sqlQuery - A string value containing SQL query
```

Comment:
Should it be  "A string value containing the SQL query" or "A string value containing an SQL query".